amplify/backend/function/fetchMotionState/src/graphql.js

```diff
@@ -81,6 +81,12 @@ module.exports = {
           hasFailed
           hasFailedNotFinalizable
           inRevealPhase
+          yaySideFullyStakedAt
```

Comment:
Really wish we wouldn't have to use this lambda. 

Pretty much this lambda single handedly  stops us from being able to display motions from uninstalled voting reputation extension contracts

That's because once the extension is uninstalled, you can no longer call state (or any other contract call) on it's contract instance. 

If we find a way to get rid of this, we would be better off.

Reply:
I would agree that this function needs at minimum a rework, a lot of the logic here we are not even using anymore (motion messages)